src/modules/passphrase/services/check.passphrase.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@nestjs/common';
```

Comment:
what is the original method? let's keep just one and remove the "copy" from the file naming